install/helm/open-match/templates/mmforc-service-account.yaml

```diff
@@ -0,0 +1,54 @@
+# Copyright 2019 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+{{- if .Values.openmatch.mmforc.install }}
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: {{ .Values.openmatch.mmforc.serviceAccount }}
+  namespace: {{ .Release.Namespace }}
+  labels:
+    app: {{ template "openmatch.name" . }}
+    chart: {{ template "openmatch.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+---
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: ClusterRoleBinding
+metadata:
+  name: {{ .Values.openmatch.mmforc.serviceAccount }}
+subjects:
+- kind: ServiceAccount
+  name: {{ .Values.openmatch.mmforc.serviceAccount }}
+  namespace: {{ .Release.Namespace }}
+roleRef:
+  kind: ClusterRole
+  name: {{ .Values.openmatch.mmforc.serviceAccount }}
+  apiGroup: rbac.authorization.k8s.io
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: {{ .Values.openmatch.mmforc.serviceAccount }}
+  namespace: {{ .Release.Namespace }}
```

Comment:
I think ClusterRole doesn't need a namespace in it's definition, it's cluster-wide: https://kubernetes.io/docs/reference/access-authn-authz/rbac/#role-and-clusterrole